External/Sparkle/bin/codesign_embedded_executable

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
It's not entirely appropriate to upgrade this file since it should be a copy of Sparkle, but I think we can close an eye given this is the only change that was required to use Python 3